db/seeds.rb

```diff
@@ -5,7 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```

Comment:
these must be references you used, Travis. Can we delete or do you still need them?

Reply:
(lines 6 -7)